scripts/CEE/describe-kafka-transaction/metadata.yaml

```diff
@@ -0,0 +1,54 @@
+file: describe-kafka-transaction.sh
+name: describe-kafka-transaction
+shortDescription: Describe the specified transaction in the Kafka instance
+description: Describe the specified transaction in the Kafka instance
+author: grdryn
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - "create"
+      apiGroups:
+        - ""
+      resources:
+        - "pods/exec"
```

Comment:
Do you need this permission at cluster scope?

Reply:
@wanghaoran1988 I _think_ so, but I'm happy to hear of any suggestions that you have. My understanding is that the script will run in the openshift-managed-scripts namespace, but for this script we pass a parameter called `resource_namespace` at run time to indicate the namespace in which the pods we need to exec to. So unless there's some magic that I don't yet understand (can we somehow provide it for `resource_namespace` dynamically?), we need to give this permission at the cluster scope so that it will have that permission in the `resource_namespace` namespace. WDYT?